src/appBootstrapper.jsx

```diff
@@ -4,6 +4,7 @@ import { Provider } from 'mobx-react';
 import { hashHistory, createMemoryHistory, Router } from 'react-router';
```

Comment:
Yes, I would really like to make this file typescript, but there are too many complications right now.